### ISSUE TITLE
Feature Request: Sync with local target directory

### ISSUE THREAD

**Issue by kj800x** [fioan89/sourcesync#40]
I have to do a lot of development on my local machine, and I need to copy the web folder from my src directory to my deploy directory every time I want my local development server to see the changes. It seems like this plugin should be able to do this easily.

**Comment by fioan89** [comment]
Yes, for the first time you can select all files from a particular folder, and sync them, after that you can either upload the current focused file, or a better solution, upload only the files that changed (this means your files are tracked by a version control software)

**Comment by kj800x** [comment]
Sorry, I didn't make myself clear.

Both my source and destination directories that I want to sync between are on my local drive (both on C:\). It seems like this plugin can only sync between a local directory and remote directory?

**Comment by fioan89** [comment]
Ouch :) yeah, I don't think it will work. In theory you can, but you need to install an ssh server and make a connection on 127.0.0.1, but I guess that's a bit of overhead, no? Can't you just make a symlink of your files in the the deploy?

**Comment by kj800x** [comment]
Yeah, I thought of that but it's too much overhead. I can't make a symlink because some of the tools we use on our source code and deploy application don't understand them.
